Projects/App/Sources/AppDelegate+Unleash.swift

```diff
@@ -27,22 +26,30 @@ extension AppDelegate {
     }
 
     private func observeUpdate() {
-        featureFlagsBag.dispose()
-        featureFlagsBag += Localization.Locale.$currentLocale
+        Localization.Locale.$currentLocale
             .atOnce()
             .distinct()
-            .onValue { locale in
+            .plain()
+            .publisher
+            .receive(on: RunLoop.main)
+            .sink { locale in
                 DispatchQueue.main.async {
                     Dependencies.featureFlags().updateContext(context: self.getContext)
                 }
             }
+            .store(in: &cancellables)
 
         let profileStore: ProfileStore = globalPresentableStoreContainer.get()
-        featureFlagsBag += profileStore.stateSignal
-            .map({ $0.memberDetails?.id })
+
+        profileStore.stateSignal
             .distinct()
-            .onValue { memberId in
+            .map({ $0.memberDetails?.id })
+            .plain()
+            .publisher
+            .receive(on: RunLoop.main)
+            .sink { memberId in
                 Dependencies.featureFlags().updateContext(context: self.getContext)
             }
+            .store(in: &cancellables)
```

Comment:
If we call setupFeatureFlags multiple times observeUpdate will be called each time.
Since we are storing observing of
```
Localization.Locale.$currentLocale
profileStore.stateSignal
```
here
`.store(in: &cancellables)
`
We are going to do observe it multiple times executing 

`Dependencies.featureFlags().updateContext(context: self.getContext)
`
multiple times

Solution would be to remove all observers from the cancellables by cancellables.removeAll() when we enter this function. Or to have seperated AnyCancellable for each of them